cli/tests/pcluster/cli/test_get_cluster_log_events.py

```diff
@@ -40,11 +42,10 @@ def test_required_args(self, args, error_message, run_cli, capsys):
     @pytest.mark.parametrize(
         "args, error_message",
         [
-            ({"start_time": "wrong"}, "Start time and end time filters must be in the ISO 8601 format"),
-            ({"end_time": "1622802790248"}, "Start time and end time filters must be in the ISO 8601 format"),
-            ({"head": "wrong"}, "invalid int value"),
-            ({"tail": "wrong"}, "invalid int value"),
-            ({"stream_period": "wrong"}, "invalid int value"),
+            ({"start_time": "wrong"}, "start_time filter must be in the ISO 8601 format"),
+            ({"end_time": "1622802790248"}, "end_time filter must be in the ISO 8601 format"),
+            ({"start_from_head": "wrong"}, "expected 'boolean' for parameter 'start-from-head'"),
+            ({"limit": "wrong"}, "expected 'number' for parameter 'limit'"),
```

Comment:
let's update the changelog to remove stream, stream-period, other removed parameters and double check the logs commands

Reply:
Can you clarify what you mean by double checking the logs commands?